unit-tests-ext/src/test/scala/scala/scalanative/junit/utils/ThrowsHelper.scala

```diff
@@ -1,15 +1,15 @@
 package scala.scalanative.junit.utils
 
-import AssertThrows._
+import AssertThrows.assertThrows
 
-// Calls to this should probably be changed to expectThrows.
+// Calls to this should probably be changed to assertThrows.
```

Comment:
This comment can probably be removed now, since it was already changed

Reply:
The thing is, I have already renamed the comment itself to remain internally consistent with the code. Before the merger the object looked like this:
```
// Calls to this should probably be changed to expectThrows.
// This was added as it was all over the place in the pre
// JUnit code.
object ThrowsHelper {
  def assertThrowsAnd[T <: Throwable, U](
      expectedThrowable: Class[T],
      code: => U)(cond: T => Boolean): Unit = {
    val c = cond(expectThrows(expectedThrowable, code))
    assert(c)
  }
}
``` 
Now that you mention it, I am not sure what this comment really means - I assumed it wanted the calls to this object to be changed, but maybe it meant just the function name (as in `expectThrowsAnd`). Perhaps it should be removed either way for the ambiguity.

Reply:
You're right, I've misunderstood the context of this comment. I belive original intent was to use `assertThrows` instead of `assertThrowsAnd`